src/main/java/wooteco/chess/controller/ChessIndexController.java

```diff
@@ -0,0 +1,20 @@
+package wooteco.chess.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+@Controller
+public class ChessIndexController {
+	@GetMapping("/")
```

Comment:
f
```suggestion
	@GetMapping
```
index는 mapping값을 생략해도 될 것 같아요 😊